node/directives/withCurrentProfile.ts

```diff
@@ -91,30 +96,40 @@ async function getCurrentProfileFromCookies(
   return null
 }
 
-async function validatedProfile(context: Context, currentProfile: CurrentProfile): Promise<CurrentProfile> {
+async function validatedProfile(
+  context: Context,
+  currentProfile: CurrentProfile
+): Promise<CurrentProfile> {
   const {
     dataSources: { profile },
   } = context
 
-  const { id, userId } = await profile.getProfileInfo(currentProfile.email, 'id')
+  const { id, userId } = await profile.getProfileInfo(
+    currentProfile.email,
+    'id'
+  )
 
   if (!id) {
     // doesn't have a profile, create one
-    await profile.updateProfileInfo(currentProfile.email, { email: currentProfile.email, userId })
+    await profile.updateProfileInfo(currentProfile.email, {
+      email: currentProfile.email,
+      userId,
+    })
   }
 
   return { userId, email: currentProfile.email }
 }
 
 function isValidCallcenterOperator(context: Context, email: string) {
   const {
-    dataSources: { callcenterOperator, licenseManager },
+    dataSources: { licenseManager },
```

Comment:
There's a native licenseManager client now :D

Reply:
Using it now